terminal/command.go

```diff
@@ -83,6 +84,8 @@ func DebugCommands(client service.Client) *Commands {
 		{aliases: []string{"stack", "bt"}, cmdFn: stackCommand, helpMsg: "stack [<depth>] [-full]. Prints stack."},
 		{aliases: []string{"frame"}, cmdFn: frame, helpMsg: "Sets current stack frame (0 is the top of the stack)"},
 		{aliases: []string{"source"}, cmdFn: c.sourceCommand, helpMsg: "Executes a file containing a list of delve commands"},
+		{aliases: []string{"on"}, cmdFn: onCmd, helpMsg: "on <breakpoint name or id> <command>. Executes command when the specified breakpoint is hit (supported commands: print, stack and goroutine without arguments)"},
```

Comment:
Let's make this more clear, something like `print OR stack / goroutine without arguments` to make it more clear that print will take an arg. Also, why limit the arg to something like stack? Seems reasonable to want to limit stack depth potentially, and since we will need to support an arg for print it doesn't seem out of the question.